pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -213,10 +197,10 @@ func (l *podNIC) PlugPhase1() error {
 			return err
 		}
 
-		if l.dhcpConfigurator != nil {
-			dhcpConfig := podNetworkingConfigurator.GenerateDHCPConfig()
+		dhcpConfig := podNetworkingConfigurator.GenerateNonRecoverableDHCPConfig()
+		if dhcpConfig != nil {
 			log.Log.V(4).Infof("The generated dhcpConfig: %s", dhcpConfig.String())
-			if err := l.dhcpConfigurator.ExportConfiguration(*dhcpConfig); err != nil {
+			if err := l.cacheFactory.CacheDHCPConfigForPid(getPIDString(l.launcherPID)).Write(l.podInterfaceName, dhcpConfig); err != nil {
```

Comment:
Would it make make sense to export this into a pod nic method ? something like `exportWhenPodInterfaceIsMutated` ? 
The name I suggest sucks, but, can be improved.